extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryPresenterCompiler.java

```diff
@@ -41,7 +43,12 @@
 import java.util.List;
 
 @SuppressWarnings({"unchecked, rawtypes"})
-final class RepositoryPresenterCompiler implements RPLayout, RPMain, RPTypedCollectionCompile {
```

Comment:
this change isn't needed, right? RPItemCompile is already implemented through RPMain?